browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -16,6 +16,32 @@
 
 /* global app $ _ L JSDialog */
 
+let _globalDecimal = '.';
+let _globalMinusSign = '-';
+
+if (typeof Intl !== 'undefined') {
+	var formatter, lang;
+	try {
+		if (app.UI.language.fromURL && app.UI.language.fromURL !== '')
+			formatter = new Intl.NumberFormat(app.UI.language.fromURL);
+		else
+			formatter = new Intl.NumberFormat(L.Browser.lang);
+
+		formatter.formatToParts(-11.1).map(function (item) {
+				switch (item.type) {
+				case 'decimal':
+					_globalDecimal = item.value;
+				break;
+				case 'minusSign':
+					_globalMinusSign = item.value;
+				break;
+			}
+		});
+	} catch(e) {
+		window.app.console.log('Exception parsing lang ' + lang + ' ' + e);
+	}
+}
```

Comment:
This is a good optimisation, but I wonder if there's a suitable object in the hierarchy that we can hang this off of? It feels a bit messy to just stick this in the global scope like this.